roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -11,56 +11,63 @@ class PackageAvailability(NotContainerizedMixin, OpenShiftCheck):
 
     def run(self, tmp, task_vars):
         rpm_prefix = get_var(task_vars, "openshift", "common", "service_type")
+        minor_version = normalized_minor_release(task_vars)
         group_names = get_var(task_vars, "group_names", default=[])
 
         packages = set()
 
         if "masters" in group_names:
-            packages.update(self.master_packages(rpm_prefix))
+            packages.update(self.master_packages(rpm_prefix, minor_version))
         if "nodes" in group_names:
-            packages.update(self.node_packages(rpm_prefix))
+            packages.update(self.node_packages(rpm_prefix, minor_version))
 
         args = {"packages": sorted(set(packages))}
         return self.module_executor("check_yum_update", args, tmp, task_vars)
 
     @staticmethod
-    def master_packages(rpm_prefix):
-        return [
-            "{rpm_prefix}".format(rpm_prefix=rpm_prefix),
-            "{rpm_prefix}-clients".format(rpm_prefix=rpm_prefix),
-            "{rpm_prefix}-master".format(rpm_prefix=rpm_prefix),
-            "bash-completion",
-            "cockpit-bridge",
-            "cockpit-docker",
-            "cockpit-kubernetes",
-            "cockpit-shell",
-            "cockpit-ws",
-            "etcd",
-            "httpd-tools",
-        ]
+    def master_packages(rpm_prefix, minor_version):
+        pkgs = {
+            "current": [
+                rpm_prefix,
+                rpm_prefix + "-clients",
+                rpm_prefix + "-master",
+                "bash-completion",
+                "cockpit-bridge",
+                "cockpit-docker",
+                "cockpit-kubernetes",
+                "cockpit-shell",
+                "cockpit-ws",
+                "etcd",
+                "httpd-tools",
+            ],
+        }
+        return pkgs[minor_version] if minor_version in pkgs else pkgs["current"]
```

Comment:
The idea is to vary the list depending on the minor version. It doesn't do that now because I haven't gathered the differences between versions yet.